EpiAware/src/latentmodels/randomwalk.jl

```diff
@@ -10,9 +10,8 @@ end
 
 @model function generate_latent(latent_model::RandomWalk, n)
     ϵ_t ~ MvNormal(ones(n))
-    σ²_RW ~ latent_model.var_prior
+    σ_RW ~ latent_model.var_prior
```

Comment:
Does this now not need a name change?

Reply:
Also note that #128 followed the lead of the random walk and used the variance vs the standard deviation. I would prefer the standard deviation formulation but think we should address in a single issue/PR change to try and maintain consistency.

Reply:
Yes that should have been an issue.

Reply:
But this stage I think increasing the scope of the PR makes more sense. I'll check for other places to switch.